28-door-state-monitor/main.py

```diff
@@ -0,0 +1,52 @@
+from machine import I2C, Pin, Timer
+import notecard 
+from notecard import card, hub, note
```

Comment:
I’m getting warnings that `card` is unused (and also `Timer` on line 1).

Reply:
I'll fix these - I suspect this was a copy paste issue and not cleaning up my imports.